subsys/net/ip/tcp2.c

```diff
@@ -34,6 +34,49 @@ static void tcp_in(struct tcp *conn, struct net_pkt *pkt);
 
 int (*tcp_send_cb)(struct net_pkt *pkt) = NULL;
 
+/* TODO: Add mutex/irq lock */
+static bool tcp_nbufs_reserve(struct tcp *conn, size_t len)
+{
+	size_t rsv_bytes = 0, rsv_bytes_old = conn->rsv_bytes;
+	bool result = false;
+	struct net_buf *buf;
+
+	while (rsv_bytes < len) {
+
+		buf = tcp_nbuf_alloc(conn, 128);
```

Comment:
why 128? and could you use a macro for it instead (with a comment summarizing its usage)